### PR TITLE
LPS-184006

### PR DESCRIPTION
**Pull request by prokofitch**
https://issues.liferay.com/browse/LPS-184006

PASSED Testing on: https://github.com/CaiocSiqueira/liferay-portal/pull/38

### CODE REVIEW

**Comment by prokofitch** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsPage.testcase [hunk 1]
```diff
@@ -895,6 +895,93 @@ definition {
...
+		}
+
+		task ("Add and select a new Publication 2") {
+			JSONPublications.addPublication(publicationName = "Publication Name 2");
+
+			JSONPublications.selectPublication(publicationName = "Publication Name 2");
+		}
+
+		task ("Add new content page with a Documents and Media widget") {
+			JSONLayout.addPublicLayout(
```
Comment (suilin) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsPage.testcase:954]: For Step 10, we should add a content page for asserting the timeline also shows up for the content page. Could you update your steps?

**Comment by prokofitch** [comment]
ci:test:sf

**Comment by suilin** [comment]
ci:forward

**Pull request closed without merging**